build-system/tasks/build.js

```diff
@@ -50,6 +51,7 @@ async function build() {
     await compileAllJs(options);
   }
   await buildExtensions(options);
+  await buildBentoComponents(options);
```

Comment:
nit: consider `await Promise.all([buildExtensions(options), buildBentoComponents(options)]);`
I'm not sure how efficiently the rest of our build systems, esbuild, closure, w/e handle running in parallel but at least this way if they do (now or eventually) we can take better advantage of it.

Reply:
I am also not sure how efficient parallel closure builds will be but it seems worth checking. If this were intend to stay for a long time I would have each of these return a set of files to be built which would then be passed to a single invocation of Closure

Reply:
for bento in particular there is no reason to begin with closure